app/src/main/res/layout/activity_nutrition_info.xml

```diff
@@ -14,20 +14,26 @@
         android:layout_height="50dp"
         android:background="@color/black"
         android:gravity="center"
-        android:text="Nutrition Info"
+        android:text="@string/nutrition_info"
         android:textColor="@color/white"
-        android:textSize="20dp"
+        android:textSize="20sp"
         android:textStyle="bold" />
 
     <SearchView
         android:id="@+id/nutritionSearch"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
+        android:layout_height="wrap_content" >
 
-    <LinearLayout
+    </SearchView>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:gravity="center">
-    </LinearLayout>
+        android:layout_height="match_parent">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="horizontal"></LinearLayout>
+    </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
Fixed the error, redesigned the page layout and added the missing strings.
Source:
https://www.youtube.com/watch?v=vXjJyLpydxg
https://www.youtube.com/watch?v=YaWmNhlOXYk